src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -82,6 +82,7 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
     onend: enterRoom,
   });
 
+  // @debt do we want to show/hide the schedule on RoomModal based on venue.showSchedule?
```

Comment:
[NB] This is a very good question, should we ask product for input or is it fine like the way it is now?

Reply:
Probably worth asking @sparkletown/product, as then we can finish it off quickly all in one PR

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1657/commits/77bee1af53d58d4907b2ec8f7d1015628a388e71 + https://github.com/sparkletown/sparkle/pull/1657/commits/a07d7ead64853134ba1b8af92f2ceb7dd05bc4a6

`venue.showSchedule` turned on
![image](https://user-images.githubusercontent.com/753891/123511595-826cc580-d6c5-11eb-8f5e-de32282a1b26.png)

`venue.showSchedule` turned off
![image](https://user-images.githubusercontent.com/753891/123511606-957f9580-d6c5-11eb-9341-1d6f7446d345.png)